app/controllers/users.js

```diff
@@ -54,3 +56,26 @@ exports.signup = (req, res, next) => {
     next(err);
   });
 };
+
+exports.signin = (req, res, next) => {
+  const user = req.body ? req.body : {};
+  emailValidation(req.body.email);
+  userService.getByEmail(user.email).then((foundUser) => {
```

Comment:
I would delegate some of this logic to the service, I mean for example I don't like the idea of the controller knowing which encryption package we are using to store the passwords.